cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsInstanceConnector.java

```
@@ -0,0 +1,177 @@
+import com.amazonaws.services.ec2.model.GetConsoleOutputRequest;
+import com.amazonaws.services.ec2.model.GetConsoleOutputResult;
+import com.amazonaws.services.ec2.model.Instance;
+import com.amazonaws.services.ec2.model.Reservation;
+import com.amazonaws.services.ec2.model.StartInstancesRequest;
+import com.amazonaws.services.ec2.model.StopInstancesRequest;
+import com.sequenceiq.cloudbreak.cloud.InstanceConnector;
+import com.sequenceiq.cloudbreak.cloud.MetadataCollector;
+import com.sequenceiq.cloudbreak.cloud.aws.view.AwsCredentialView;
+import com.sequenceiq.cloudbreak.cloud.event.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.CloudVmInstanceStatus;
+import com.sequenceiq.cloudbreak.cloud.model.InstanceStatus;
+
+@Service
+public class AwsInstanceConnector implements InstanceConnector {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsInstanceConnector.class);
+
+    @Inject
+    private AwsClient awsClient;
+
+    @Inject
+    private AwsMetadataCollector awsMetadataCollector;
+
+    @Override
+    public String getConsoleOutput(AuthenticatedContext authenticatedContext, CloudInstance vm) {
+        AmazonEC2Client amazonEC2Client = awsClient.createAccess(new AwsCredentialView(authenticatedContext.getCloudCredential()),
+                authenticatedContext.getCloudContext().getRegion());
+        GetConsoleOutputRequest getConsoleOutputRequest = new GetConsoleOutputRequest().withInstanceId(vm.getInstanceId());
+        GetConsoleOutputResult getConsoleOutputResult = amazonEC2Client.getConsoleOutput(getConsoleOutputRequest);
+        try {
+            if (getConsoleOutputResult.getOutput() == null) {
+                return "";
+            } else {
+                return getConsoleOutputResult.getDecodedOutput();
+            }
+        } catch (Exception ex) {
+            return "";
```

exeception is silently ignored